app/Console/Commands/AbstractInitializeCommand.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Data\DataIdentifier;
+use DateInterval;
+use DateTime;
+use Illuminate\Console\Command;
+
+class AbstractInitializeCommand extends Command
```

Comment:
```suggestion
abstract class AbstractInitializeCommand extends Command
```
if classname contains abstract is should be an actual abstract class